cylc/flow/etc/job.sh

```diff
@@ -96,28 +96,43 @@ cylc__job__main() {
     else
         CYLC_TASK_WORK_DIR_BASE="${CYLC_TASK_CYCLE_POINT}/${CYLC_TASK_NAME}"
     fi
-    export CYLC_TASK_WORK_DIR="${CYLC_SUITE_WORK_DIR}/${CYLC_TASK_WORK_DIR_BASE}"
-    typeset contact="${CYLC_SUITE_RUN_DIR}/.service/contact"
+    export CYLC_TASK_WORK_DIR="${CYLC_WORKFLOW_WORK_DIR}/${CYLC_TASK_WORK_DIR_BASE}"
+    typeset contact="${CYLC_WORKFLOW_RUN_DIR}/.service/contact"
     if [[ -f "${contact}" ]]; then
-        CYLC_SUITE_HOST="$(sed -n 's/^CYLC_SUITE_HOST=//p' "${contact}")"
-        CYLC_SUITE_OWNER="$(sed -n 's/^CYLC_SUITE_OWNER=//p' "${contact}")"
-        export CYLC_SUITE_HOST CYLC_SUITE_OWNER
+        CYLC_WORKFLOW_HOST="$(sed -n 's/^CYLC_WORKFLOW_HOST=//p' "${contact}")"
+        CYLC_WORKFLOW_OWNER="$(sed -n 's/^CYLC_WORKFLOW_OWNER=//p' "${contact}")"
+        export CYLC_WORKFLOW_HOST CYLC_WORKFLOW_OWNER
     fi
     # DEPRECATED environment variables
-    export CYLC_SUITE_SHARE_PATH="${CYLC_SUITE_SHARE_DIR}"
-    export CYLC_SUITE_INITIAL_CYCLE_TIME="${CYLC_SUITE_INITIAL_CYCLE_POINT}"
-    export CYLC_SUITE_FINAL_CYCLE_TIME="${CYLC_SUITE_FINAL_CYCLE_POINT}"
+    export CYLC_WORKFLOW_SHARE_PATH="${CYLC_WORKFLOW_SHARE_DIR}"
+    export CYLC_WORKFLOW_INITIAL_CYCLE_TIME="${CYLC_WORKFLOW_INITIAL_CYCLE_POINT}"
+    export CYLC_WORKFLOW_FINAL_CYCLE_TIME="${CYLC_WORKFLOW_FINAL_CYCLE_POINT}"
```

Comment:
If these are deprecated, can't the left-hand side be left as is (and the corresponding entries below removed)?

Reply:
Oops, yeah, looks like a `sed` fail (user error).